HTLV, HCV and DENV tools

murilofreire opened the pull request:
Hi,
Vagner, Ines and myself finished the implementation of HTLV 1 and 2, HCV and DENV tools on rega genotype. I created a new branch called fiocruz to allocate this changes. I've downloaded this source code and tested on eclipse and I think everything is working. Please analyse and merge.

Kind regards,

Murilo Freire

murilofreire committed:
HTLV, HCV and DENV tools

kdeforche commented:
Hey, it looks good but you've added a lot of files that shouldn't be there: .metadata/_, .settings/_ and also what seem to be files from testing things (Blast databases, runs, etc...). Can you remove those and update your pull request?

murilofreire commented:
OK, I'll do that.

R,

Murilo

2015-06-26 7:41 GMT-03:00 Koen Deforche notifications@github.com:

> Hey, it looks good but you've added a lot of files that shouldn't be
> there: .metadata/_, .settings/_ and also what seem to be files from
> testing things (Blast databases, runs, etc...). Can you remove those and
> update your pull request?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/rega-cev/rega-genotype/pull/1#issuecomment-115638953.

murilofreire commented:
Koen,

I removed the .metadata, .settings, RemoteSystemsTempFiles. I didn't follow
what are the xml files you are asking me to remove from commit. We modified
the web.xml because we fixed some issues. Are you referring to the files
resulting of jobs?

Kind regards

Murilo

2015-06-26 9:40 GMT-03:00 Murilo Freire murilofreire666@gmail.com:

> OK, I'll do that.
> 
> R,
> 
> Murilo
> 
> 2015-06-26 7:41 GMT-03:00 Koen Deforche notifications@github.com:
> 
> > Hey, it looks good but you've added a lot of files that shouldn't be
> > there: .metadata/_, .settings/_ and also what seem to be files from
> > testing things (Blast databases, runs, etc...). Can you remove those and
> > update your pull request?
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/rega-cev/rega-genotype/pull/1#issuecomment-115638953
> > .
